documentation/applying_manifest_blocks.md

```diff
@@ -246,7 +246,9 @@ You can change this with the `hiera-config` key in a Bolt config file or the
 
 Following the Hiera 5 convention, the default data dir is relative to
 `hiera.yaml` at `$BOLTDIR/data`. For configfile examples, see [Configuring
```

Comment:
```suggestion
`hiera.yaml` at `$BOLTDIR/data`. For configuration file examples, see [Configuring
```

Reply:
When we use `$BOLTDIR` here, are we talking specifically about an embedded project directory, or is $BOLTDIR an environment variable? If we're just talking about a Bolt project directory, then maybe we can just use <PROJECT_DIRECTORY>?

Reply:
No, I think it's what we used to use for `<project_directory>`. Should it be all caps?

Reply:
It looks like elsewhere we use PROJECT_NAME https://puppet.com/docs/bolt/latest/bolt_running_plans.html#how-bolt-locates-plans

Reply:
And <project-directory> https://puppet.com/docs/bolt/latest/configuring_bolt.html#bolt-projectyaml

Reply:
Good point. Let's standardize it as `<PROJECT_DIRECTORY>` so we're following Puppet style. We should change `<PROJECT_NAME>` anyway, because technically your project directory name and project name don't have to match.

Reply:
Ended up with `<PROJECT DIRECTORY>` which seems clearest and inline with style guidelines.